ctms/acoustic_service.py

```diff
@@ -233,27 +233,20 @@ def _newsletter_converter(self, acoustic_main_table, contact, newsletters_mappin
             acoustic_main_table[sub_flag] = "0"
 
         for newsletter in contact_newsletters:
-            newsletter_template = {
+            newsletter_row = {
                 "email_id": contact_email_id,
+                "newsletter_name": newsletter.name,
+                "newsletter_source": newsletter.source and str(newsletter.source),
+                "create_timestamp": newsletter.create_timestamp.date().isoformat(),
+                "update_timestamp": newsletter.update_timestamp.date().isoformat(),
                 "newsletter_format": contact_email_format,
                 "newsletter_lang": contact_email_lang,
```

Comment:
Might this be the right time to save these rows with `newsletter.format` and `newsletter.lang` 😈 

https://github.com/mozilla-it/ctms-api/blob/0456cbdaaa660512bc2c2d0935d642f67cdd9616/ctms/models.py#L113-L114